### PR TITLE
xTicks + yTicks default to using pretty()

### PR DESCRIPTION
**Pull request by lindsaycarr**
Fixing #95. 

@ldecicco-USGS is this all you are talking about?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ install.packages("EGRET")
     ## package 'EGRET' successfully unpacked and MD5 sums checked
     ## 
     ## The downloaded binary packages are in
-    ##  C:\Users\lcarr\AppData\Local\Temp\1\RtmpAPk7Z0\downloaded_packages
+    ##  C:\Users\lcarr\AppData\Local\Temp\1\Rtmp8QoZAB\downloaded_packages
```
Comment (ldecicco-USGS) [inline README.md]: what do you think, should we keep in the messages (it's kind of nice to see what the real output looks like)? Also a sneaky way to make sure people are rebuilding the readme

Comment (lindsaycarr) [inline README.md]: So keep `messages=TRUE` for code chunks is what you are thinking? It's nice, but does make it look like my changes impacted the README when they really didn't.

Comment (ldecicco-USGS) [inline README.md]: yeah, I'm just saying I like the way it is, and it will change every time someone re-knit's the readme...but, so what? agree?

Comment (lindsaycarr) [inline README.md]: OK, yes - I agree with that

**Pull request merged**